### PR TITLE
Reframe Buttons and Cards project

### PR DESCRIPTION
**Pull request by timkinnane**
@diegodorgam please review to see if it's still inline with your idea of the project. I've tried to keep the requirements quite specific but expand the vision, to hopefully be more attractive to students.

### CODE REVIEW

File: contributing/google-summer-of-code-2018/README.md [hunk 1]
```diff
@@ -176,11 +176,12 @@ Interested students are also encouraged to interact with our contributor communi
 
 - **Desirable Skills:** Familiarity with JavaScript and Python development.
 
-### Buttons and Cards UI
+### Rich Messages and Payloads
```
Comment (diegodorgam) [inline contributing/google-summer-of-code-2018/README.md]: I think we should keep buttons and cards reference in the title. Like `Rich Messages and Payloads for Buttons and Cards`

File: contributing/google-summer-of-code-2018/README.md [hunk 2]
```diff
@@ -176,11 +176,12 @@ Interested students are also encouraged to interact with our contributor communi
...
 
 - **Mentors:** @diego.dorgam + @timkinnnae
-- **Description:** Include the possibility to send a message with customizable buttons and cards carousel (slidable images with text descriptions) so the user can interact with and bots can receive the feedback of those interactions, right on the Rocket.Chat's interface, and compatible (if possible) with existing UI conventions, so it becomes intuitive to their community to develop to Rocket.Chat.  
-- **Desirable Skills:** Familiarity with JavaScript development.
+- **Description:** Create UI components and develop the handling of message payloads to support new interactions for bots, like buttons, cards, images and more. Follow existing conventions to let bot developers migrate pre-built solutions to be delivered via Rocket.Chat. This opens the door for enhanced user experience and interaction flows in automated messaging, letting bots provide a much greater variety of utility; such as business apps, team workflows, even games and interactive narratives.
```
Comment (diegodorgam) [inline contributing/google-summer-of-code-2018/README.md:182]: awesome, much better! thanks @timkinnane

**Review by MartinSchoeler — APPROVED** [review]

**Comment by MartinSchoeler** [comment]
Waiting on @diegodorgam approval to merge

File: contributing/google-summer-of-code-2018/README.md (hunk 1 above)
Comment (timkinnane) [reply contributing/google-summer-of-code-2018/README.md]: @diegodorgam sorry looks good to me and I've been meaning to action this but I just saw it got pulled without that. Should we do another PR to change it as you suggested?

**Pull request merged**